src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -289,53 +258,21 @@ def __await__(self) -> ResponseType:
         return response
 
 
-class UnaryStreamCall(Call, _base_call.UnaryStreamCall):
-    """Object for managing unary-stream RPC calls.
-
-    Returned when an instance of `UnaryStreamMultiCallable` object is called.
-    """
-    _request: RequestType
-    _metadata: MetadataType
-    _request_serializer: SerializingFunction
-    _response_deserializer: DeserializingFunction
-    _send_unary_request_task: asyncio.Task
+class _StreamResponseMixin(Call):
     _message_aiter: AsyncIterable[ResponseType]
+    _prerequisite: asyncio.Task
```

Comment:
I do not like the `_prerequisite` name, but I do not have great alternatives just names like `_previous_call` or `_first_call` ...